src/integration/integration_test.go

```diff
@@ -5,23 +5,32 @@ import (
 	"context"
 	"embed"
 	"fmt"
+	"github.com/google/go-cmp/cmp"
+	"github.com/onsi/gomega/gbytes"
+	v1 "go.opentelemetry.io/proto/otlp/resource/v1"
```

Comment:
Although it's not absolutely necessary, external libraries are typically grouped separately in the import list from standard libraries. Can you check out and apply [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports)?